controllers/users.js

```diff
@@ -48,6 +51,14 @@ const getTableActions = (item, path) => {
 			title: 'Registrierungslink generieren',
 		});
 	}
+	if (USER_MIGRATION_ENABLED) {
```

Comment:
I would prefer if its deactivated when the user cant be rollbacked, because he wasnt migrated before. Otherwise it would suggest actions which the superhero cant perform.
We can talk with bb if you want

Reply:
it was requested by boris that the dialog can be opened for every user, so he can still see the data.